tools/coqdoc/cpretty.mll

```diff
@@ -131,12 +138,18 @@
     Cdglobals.light := s.st_light
 
   let begin_show () = save_state (); Cdglobals.gallina := false; Cdglobals.light := false
-  let end_show () = restore_state ()
+  let end_show lexbuf =
+    try restore_state ()
+    with Stack.Empty -> eprintf "%a, Error: ill-bracketed 'end show'\n" print_position lexbuf
```

Comment:
This seems like a bad idea. The execution will continue as if nothing wrong happened. At the very least, the exception should be reraised of `exit 1` should be called.